src/operator/OrgOverlay.tsx

```diff
@@ -57,6 +57,7 @@ const OrgOverlay: FC = () => {
   const history = useHistory()
   const canReactivateOrg =
     hasWritePermissions && organization?.state === 'suspended'
+  const canSeeCardinalityLimits = organization?.storageType == 'tsm'
```

Comment:
I think we might want to invert this conditional so that we greedily show cardinality limits. From what I understand, every org _should_ have a `storageType` associated with it, but there might have been some orgs that fell through the cracks, and if they are tsm orgs without a `storageType`, this won't show cardinality limits properly. Probably a longshot edge case, but worth considering.

Example of what I'm rambling about:
```ts
import {isOrgIOx} from 'src/organizations/selectors'
// ...

const isIOx = useSelector(isOrgIOx)
const canSeeCardinalityLimits = !isIOx
```